src/FerriteGmsh.jl

```diff
@@ -60,6 +99,111 @@ function translate_elements(original_elements::Vector{Cell{3,20,6}})
                                              original_ele.nodes[16])),)
     end
     return ferrite_elements
+end
+
+
+#  y
+#
+#  ^
+#  |
+#  |
+#  +--- > x
+#  \\
+#   \\
+#    \\
+#     z
+#
+#     GMSH
+# 4----14----3
+# |\         |\
+# |16    25  | 15
+# 10 \ 21    12 \
+# |   8----20+---7
+# |23 |  27  | 24|
+# 1---+-9----2   |
+#  \ 18    26 \  19
+#  11 |  22    13|
+#    \|         \|
+#     5----17----6
+#
+#    Ferrite
+# 4----11----3
+# |\         |\
+# |20    24  | 19
+# 12 \ 21    10 \
+# |   8----15+---7
+# |25 |  27  | 23|
+# 1---+-9----2   |
+#  \ 16     26\  14
+#  17 |  22    18|
+#    \|         \|
+#     5----13----6
+#
+Ferrite.faces(c::Cell{3,27,6}) = ((c.nodes[1],c.nodes[4],c.nodes[3],c.nodes[2]), (c.nodes[1],c.nodes[2],c.nodes[6],c.nodes[5]), (c.nodes[2],c.nodes[3],c.nodes[7],c.nodes[6]), (c.nodes[3],c.nodes[4],c.nodes[8],c.nodes[7]), (c.nodes[1],c.nodes[5],c.nodes[8],c.nodes[4]), (c.nodes[5],c.nodes[6],c.nodes[7],c.nodes[8]))
+Ferrite.cell_to_vtkcell(::Type{Cell{3, 27, 6}}) = WriteVTK.VTKCellTypes.VTK_TRIQUADRATIC_HEXAHEDRON
+Ferrite.nodes_to_vtkorder(cell::Cell{3, 27, 6}) = [
+    cell.nodes[1], # faces
+    cell.nodes[2],
+    cell.nodes[3],
+    cell.nodes[4],
+    cell.nodes[5],
+    cell.nodes[6],
+    cell.nodes[7],
+    cell.nodes[8],
+    cell.nodes[9], # edges
+    cell.nodes[10],
+    cell.nodes[11],
+    cell.nodes[12],
+    cell.nodes[13],
+    cell.nodes[14],
+    cell.nodes[15],
+    cell.nodes[16],
+    cell.nodes[17],
+    cell.nodes[18],
+    cell.nodes[19],
+    cell.nodes[20],
+    cell.nodes[25], # faces
+    cell.nodes[23],
+    cell.nodes[22],
+    cell.nodes[24],
+    cell.nodes[21],
+    cell.nodes[26],
+    cell.nodes[27], # interior
+]
```

Comment:
this should go into ferrite right?

Reply:
Yes, but it is broken in 0.3.14, so this is the fix for it. Newer releases will need https://github.com/Ferrite-FEM/Ferrite.jl/pull/714/ to be merged.

Reply:
but the methods are already dispatched in Ferrite, aren't they? So this results maybe in undefined behavior due to hardcore type piracy?

Reply:
How to do it then otherwise?

Reply:
I'm not sure if it's possible at all if that can really lead to undefined behavior but I don't know. I just know the hint in the docs. I cannot say how serious this problem is

Reply:
Whelp @fredrikekre :D

Reply:
I guess it worked for you on linux, don't know if you can fuck up things if you load the packages differently. So, I guess you loaded Ferrite and then FerriteGmsh, does it still work vice verca? And my main concern is somewhat Windows, because Windows

Reply:
If I load `FerriteGmsh` then `Ferrite` is loaded anyway, right? And Windows is not working for several months now anyway, so I do not understand this part.

Reply:
Ferrite is then loaded before you overwrite the methods. If you load the Ferrite definitions afterwards, I'm not sure if it still works. Windows issue is a binary problem and not related to piracy or anything functional wrong within the package